Assignment/solution_student/select_fun_part2/js/main.js

```diff
@@ -1,29 +1,55 @@
 // 1.	Slide the header up when it is clicked on.
+$("header").on("click", function(){
+	$("header").slideUp(400);
+});
 
 // 2.	Remove the purplebox class
+$(".box").removeClass("purplebox");
 
 // 3.	Hide all of the div’s in #row4
+$("#row4").hide();
```

Comment:
_Close!_ It was actually looking for _just_ the `div`s in #row4, not the whole row. You got the point though :)